src/modules/formBuilder/components/itemEditor/pickLists/PickListOption.tsx

```diff
@@ -0,0 +1,111 @@
+import { Stack } from '@mui/material';
+import { useCallback } from 'react';
+import ControlledCheckbox from '@/modules/form/components/rhf/ControlledCheckbox';
+import ControlledTextInput from '@/modules/form/components/rhf/ControlledTextInput';
+import { FormItemControl } from '@/modules/formBuilder/components/itemEditor/types';
+import { Component } from '@/types/gqlTypes';
+
+interface PickListOptionProps {
+  control: FormItemControl;
+  index: number;
+  formItemComponent: Component;
+  setPickListInitialSelected: (code: string) => void;
+  isCodeUnique: (coe: string) => boolean;
+  code?: string;
+}
+
+const PickListOption: React.FC<PickListOptionProps> = ({
+  control,
+  code,
+  index,
+  formItemComponent,
+  isCodeUnique,
+  setPickListInitialSelected,
+}) => {
+  const onChangeInitialSelected = useCallback(
+    (value: boolean) => {
+      // Even though it's a controlled RHF checkbox, we still provide an onChange callback
+      // in order to set all the other pick list options' `initialSelected` values to false when this one is set to true.
+      if (value && code) setPickListInitialSelected(code);
```

Comment:
I think there is an existing bug with initial selected, but I'm considering it out of the scope of this ticket; I've created a bug ticket that links to a form that reproduces the bug on QA. https://github.com/open-path/Green-River/issues/6368

Reply:
Thanks for making a ticket!